src/app/sitemap.ts

```diff
@@ -40,5 +45,19 @@ export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
     },
   );
 
-  return [...routes, ...articles];
+  // swap pages
+  const { chains } = await getChainsQuery();
+  const swapPages = chains.map((chain) => {
+    return {
+      url: withoutTrailingSlash(
+        `${getSiteUrl()}${JUMPER_SWAP_PATH}/${chain.name}`
+          .replace(' ', '-')
+          .toLowerCase(),
+      ),
+      lastModified: new Date().toISOString().split('T')[0],
+      priority: 0.4,
+    };
+  });
+
+  return [...routes, ...articles, ...swapPages];
```

Comment:
@tcheee @oktapodia 

Since, we are only talking about a few dozen entries of those swap-pages, I did not copy the logic from what @oktapodia did on the bridge pages. Instead, it´s sitting in here. Wdyt?